.gitignore

```diff
@@ -80,3 +80,35 @@ ircmaxell/random-lib/composer.lock
 ircmaxell/random-lib/phpunit.xml.dist
 
 icewind/streams/tests
+
+guzzlehttp/guzzle/build/
+guzzlehttp/guzzle/docs/
+guzzlehttp/guzzle/tests/
+guzzlehttp/guzzle/.gitignore
+guzzlehttp/guzzle/.travis.yml
+guzzlehttp/guzzle/CHANGELOG.md
+guzzlehttp/guzzle/composer.json
+guzzlehttp/guzzle/LICENSE
```

Comment:
*sigh* – yeah - will add once Jenkins is somewhat happy with https://github.com/owncloud/core/pull/14913